app/pages/search/filter.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { injectT } from 'i18n';
+import SelectControl from './controls/SelectControl';
+import * as filterActions from '../../actions/filterActions';
+
+class Filter extends Component {
```

Comment:
This is not a filter it should be something like `SearchSort` under its own folder: `search/sort/SearchSort.js` so it would follow the current naming/foldering.